luaui/Widgets/cmd_extractor_snap.lua

```diff
@@ -0,0 +1,209 @@
+function widget:GetInfo()
+	return {
+		name      = "Extractor Snap (mex/geo)",
+		desc      = "Snaps extractors to give nearest spot",
+		author    = "Hobo Joe, based on work by Niobium and Floris",
+		version   = "v1.0",
+		date      = "Jan 2024",
+		license   = "GNU GPL, v2 or later",
+		layer     = 1,
+		enabled   = true,
+		handler   = true
```

Comment:
needed?

Reply:
Good chance it is not! I inherited it from the widgets this is based on. I'm not entirely sure what `handler` does here.